cpp/unit/unit_texture.cpp

```diff
@@ -145,7 +145,7 @@ unsigned int dir_group(coord::phys3_delta dir, unsigned int angles) {
 	unsigned int first_angle = 5 * angles / 8;
 
 	// normalise direction vector
-	double len = std::hypot(dir.ne, dir.se);
+	double len = std::hypot(dir.ne, dir.se).to_double();
```

Comment:
But _which_ FixedPoint should we return? If you e.g. divide a FixedPoint<int, 256> by a FixedPoint<int, 256>, should we return a FixedPoint<int, 256>?